examples/getstarted/config/database.js

```diff
@@ -9,10 +9,10 @@ const sqlite = {
 const postgres = {
   client: 'postgres',
   connection: {
-    database: 'strapi',
-    user: 'strapi',
-    password: 'strapi',
-    port: 5432,
+    database: 'postgres',
+    user: 'postgres',
+    password: 'test',
+    port: 54321,
```

Comment:
I would prefer to keep the old credentials, as all databases sahre the same ones! is there any reason to change these ones?